src/plugins/home/public/application/components/guided_onboarding/getting_started.tsx

```diff
@@ -86,15 +99,18 @@ export const GettingStarted = () => {
           </EuiText>
           <EuiSpacer size="s" />
           <EuiSpacer size="xxl" />
-          <EuiFlexGrid columns={3} gutterSize="xl">
+          <EuiFlexGrid columns={4} gutterSize="l">
```

Comment:
I was unable to insert directly on line `97` but can we slightly modify the line to read: `<EuiText size="m" textAlign="center">`